sklearn/model_selection/tests/test_search.py

```diff
@@ -814,6 +814,12 @@ def test_grid_search_cv_results():
     n_grid_points = 6
     params = [dict(kernel=['rbf', ], C=[1, 10], gamma=[0.1, 1]),
               dict(kernel=['poly', ], degree=[1, 2])]
+    grid_search = GridSearchCV(SVC(gamma="scale"), cv=n_splits, iid=False,
+                               param_grid=params)
+    grid_search.fit(X, y)
+    grid_search_iid = GridSearchCV(SVC(gamma="scale"), cv=n_splits, iid=True,
```

Comment:
You don't need gamma='scale' here and below where gamma is being searched over: the SVC will never be fit with gamma='auto_deprecated'

Reply:
I don't understand as to why this changes isn't needed. If we don't use `gamma='scale'` here explicitly, then this will raise a warning, which isn't desired considering https://github.com/scikit-learn/scikit-learn/pull/10331#issuecomment-353168653.